weave/tests/test_plot.py

```diff
@@ -8,6 +8,19 @@
 from .test_run_segment import create_experiment
 from .. import storage
 
+from contextlib import contextmanager
+
+
+@contextmanager
+def const_nodes_equal():
```

Comment:
i love the creativity - but why not just make a helper function?

Reply:
`==` already implements all of the recursive logic for comparing lists, dicts, and other nested structures. by doing it this way we can make use of all of that logic, without having to reimplement it, and just change the way const nodes operate under `==`. for example, if I did something like:

```
def const_nodes_equal(val1, val2): 
   ...
```

i would have to check if val1 is a list, then map over the list, check if its a dict, map over the dict, etc. this way i can get all that for free and just update the logic in the one place i want